lib/ui/Urlbar.js

```diff
@@ -58,11 +58,7 @@ Urlbar.prototype = {
 
     if (evt.ctrlKey || evt.altKey || evt.metaKey) {
       // special keys could mean a hotkey combination, so bail
-      // TODO: extract port from popup so that urlbar can use it too
```

Comment:
Since the popup is now listening for the 'popupshowing' and 'popuphiding' events fired by gecko, we don't need to duplicate these popupclose / popupopen calls.